<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profilometer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>139</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Laser Scan MIcrometer</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="ScanParameters">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Scan Parameters</string>
    </property>
    <widget class="QPushButton" name="btn_SCAN">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SCAN</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_Npoints">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_Nsamples">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Samples</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_sampleLength">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Interval (mm)</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_microns">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stop">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop SCAN</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>66</width>
       <height>24</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string> Motion control distance along axis (mm)</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_go1">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="sb_goto1">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTipDuration">
        <number>2</number>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>300.000000000000000</double>
       </property>
       <property name="value">
        <double>60.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_goto2">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="sb_goto2">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>300.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btn_goto3">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="sb_goto3">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>300.000000000000000</double>
       </property>
       <property name="value">
        <double>125.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btn_goto3_2">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="sb_goto3_2">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>300.000000000000000</double>
       </property>
       <property name="value">
        <double>150.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbl_current_pos">
       <property name="text">
        <string>Current Position</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="curr_position">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>300.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>681</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_left">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>47</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_right">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>47</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>500</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set X-range</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>81</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSpinBox" name="sb_lower_limit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>-10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_upper_limit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_limit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Limits</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_Clear">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>670</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>81</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSpinBox" name="sb_lower_limit_intensity">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_upper_limit_intensity">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>50</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="value">
      <number>250</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_limit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Limits</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="comments">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>540</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="comment_label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>510</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Comments</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProfilometer">
    <property name="title">
     <string>Profilometer</string>
    </property>
    <addaction name="action_File"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Save"/>
    <addaction name="actionLoad"/>
    <addaction name="action_Print"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menuTIlt_Correction">
    <property name="title">
     <string>TIlt Correction</string>
    </property>
    <addaction name="actionTilt_Correct"/>
   </widget>
   <addaction name="menuProfilometer"/>
   <addaction name="menu_File"/>
   <addaction name="menuTIlt_Correction"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_File">
   <property name="text">
    <string>&amp;File</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionTilt_Correct">
   <property name="text">
    <string>Tilt Correct</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="text">
    <string>&amp;Print</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
